operator/src/test/java/oracle/kubernetes/operator/steps/ManagedServersUpStepTest.java

```diff
@@ -795,9 +795,11 @@ static Collection<String> getPreCreateServers() {
     }
 
     static ServerStartupInfo getServerStartupInfo(String serverName) {
-      for (ServerStartupInfo startupInfo : info.getServerStartupInfo()) {
-        if (startupInfo.serverConfig.getName().equals(serverName)) {
-          return startupInfo;
+      if (info.getServerStartupInfo() != null) {
```

Comment:
This can be simplified by extracting a method that does what the original ``getServerStartupInfo`` did: use Optional.ofNullable() to always return a collection.